.gitignore

```diff
@@ -0,0 +1 @@
+/vendor/
```

Comment:
Re the below comment, let's remove this gitignore file. Depending how you manage your dependencies, this makes some assumptions?